selfhosted

```diff
@@ -11,7 +11,8 @@ cd "$TMP_DIR/tests"
 
 source /nfs/users/github/actions-runner/_work/qibojit/qibojit/testenv/bin/activate
 
-pytest
+pytest --gpu_only
```

Comment:
I forgot: to avoid crossing fingers while typing, I'd suggest to stick to only one between `-` and `_`, whenever possible
```suggestion
pytest --gpu-only
```